calico_containers/pycalico/datastore.py

```diff
@@ -134,7 +134,15 @@ class DatastoreClient(object):
 
     def __init__(self):
         etcd_authority = os.getenv(ETCD_AUTHORITY_ENV, ETCD_AUTHORITY_DEFAULT)
-        (host, port) = etcd_authority.split(":", 1)
+        try:
+            (host, port) = etcd_authority.split(":", 1)
```

Comment:
I don't think this will catch if the authority is ":1234", but I think it will be pretty obvious when it fails elsewhere..